src/main/java/trivia/QuestionCategoryName.java

```diff
@@ -0,0 +1,24 @@
+package trivia;
+
+enum QuestionCategoryName {
+
+  POP("Pop"),
+  SCIENCE("Science"),
+  SPORTS("Sports"),
+  ROCK("Rock");
+
+  private final String name;
+
+  QuestionCategoryName(String name) {
+    this.name = name;
+  }
+
+  public QuestionCategory createQuestionList() {
+    return QuestionCategory.create(this, 50);
```

Comment:
Number of questions is knowledge of the Game control class, not buried in the enum.